pkg/devicemapper/devmapper.go

```diff
@@ -284,9 +284,9 @@ func FindLoopDeviceFor(file *os.File) *os.File {
 	return nil
 }
 
-func UdevWait(cookie uint) error {
-	if res := DmUdevWait(cookie); res != 1 {
-		logrus.Debugf("Failed to wait on udev cookie %d", cookie)
+func UdevWait(cookie *uint) error {
```

Comment:
Yes. That's the reason. We want to make sure we get the final value of cookie (as set by libdm calls) in deferred function.